mindmeld/components/dialogue.py

```diff
@@ -805,11 +805,11 @@ def _validate(self, request, slot):
             if entity_type in DEFAULT_SYS_ENTITIES:
                 # system entity validation - checks for presence of required system entity
 
-                entity_text = (
-                    str(request.entities[0]["value"][0]["value"])
-                    if request.entities
-                    else text
-                )
+                try:
```

Comment:
can you add the tests that this try-except is supposed to catch?

Reply:
Added!